newton/examples/robot/example_robot_g1.py

```
@@ -140,7 +140,7 @@ def test(self):
             self.model,
             self.state_0,
             "all body velocities are small",
-            lambda q, qd: max(abs(qd)) < 0.001,
+            lambda q, qd: max(abs(qd)) < 0.005,
```

Ruff (0.14.0)

Unused lambda argument: `q`

(ARG005)